config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'auth/callback'
+  get 'auth/failure'
```

Comment:
Do these need to be defined here still - it looms like lines 17 and 18 are covering these routes